docs/root/faq/api/control_plane.rst

```diff
@@ -0,0 +1,28 @@
+How do I support multiple xDS API major versions in my control plane?
+=====================================================================
+
+Where possible, it is highly recommended that control planes support a single major version at a
+given point in time for simplicity. This works in situations where control planes need to only
+support a window of Envoy versions which spans less than a year.
+
+For control planes that need to support a wider range of versions, there are a few approaches:
+
+1. Independent v2/v3 configuration generation pipelines. This is simple to understand but
+   involves significant duplication of code and can be expensive engineering wise. This may
+   work well if the API surface in use is small.
+2. Have the control plane use v2 canonically and mechanically transform v2 messages to their
```

Comment:
From an experiment [here](https://github.com/envoyproxy/go-control-plane/pull/290) it looks like we need some fixes in envoy to accept mixed responses.

Reply:
may be im wrong. Taking a closer look again to debug.

Reply:
Couldn't get it working yet. 
Error after some more fixes to control plane code:
```
[2020-04-17 19:18:00.722][1209][debug][config] [source/common/config/grpc_mux_impl.cc:136] Received gRPC message for type.googleapis.com/envoy.config.cluster.v3.Cluster at version v0
[2020-04-17 19:18:00.722][1209][warning][config] [source/common/config/grpc_mux_impl.cc:139] Ignoring the message for type URL type.googleapis.com/envoy.config.cluster.v3.Cluster as it has no current subscribers.
```